be/src/exec/pipeline/pipeline_driver_dispatcher.cpp

```diff
@@ -61,8 +61,8 @@ void GlobalDriverDispatcher::run() {
 
         auto* query_ctx = driver->query_ctx();
         auto* fragment_ctx = driver->fragment_ctx();
-        // This writing is to ensure that MemTracker will not be destructed before the thread ends.
-        // This writing method is a bit tricky, and when there is a better way, replace it
+        // TODO(trueeyu): This writing is to ensure that MemTracker will not be destructed before the thread ends.
```

Comment:
Why trueeyu?

Reply:
its a transient solution proposed by trueeyu, so TODO is added to hint trueeyu to resolve this problem finally and uniformly when a good solution is elaborated.